### ISSUE TITLE
Is there a version compatible with winston 3.0.x?

### ISSUE THREAD

**Issue by ADumaine** [winstonjs/winston-redis#28]

**Comment by DABH** [comment]
A winston@3-compatible version now exists on the `master` branch thanks to @mgibeau , and hopefully this is published to npm soon!  Thanks!

**Comment by jairelton** [comment]
No plans to publish a new version?

**Comment by mgibeau** [comment]
Yup, sorry, this took a bit of time to sort out, I just released a `beta` tag.

**Comment by occultskyrong** [comment]
Will the new version be released?

**Comment by mgibeau** [comment]
Hi, there is a version available that is compatible with Winston 3.0.x, you can see it here: https://www.npmjs.com/package/winston-redis/v/3.0.1.

It is still tagged as beta as I could not guarantee all use case are functional and a patch had to be introduced because of a regression in the newer versions of Winston.
